crates/parcel/src/parcel.rs

```diff
@@ -28,4 +30,14 @@ impl Parcel {
       rpc: options.rpc,
     }
   }
+
+  pub fn build(&self) -> anyhow::Result<()> {
+    let mut _rpc_connection = None::<RpcConnectionRef>;
+
+    if let Some(rpc_host) = &self.rpc {
+      _rpc_connection = Some(rpc_host.start()?);
+    }
+
+    Ok(())
+  }
```

Comment:
This initializes workers only once the build has started. The workers use channels for transferring data so they can be initialized lazily as their messages are buffered. Currently the build blocks on their initialization